cmd/cloud.go

```diff
@@ -345,27 +357,20 @@ This will execute the test on the k6 cloud service. Use "k6 login cloud" to auth
 		},
 	}
 	cloudCmd.Flags().SortFlags = false
-	cloudCmd.Flags().AddFlagSet(cloudCmdFlagSet(globalFlags))
+	cloudCmd.Flags().AddFlagSet(cloudCmdFlagSet(&showCloudLogs, &exitOnRunning))
 	return cloudCmd
 }
 
-func cloudCmdFlagSet(globalFlags *commandFlags) *pflag.FlagSet {
+func cloudCmdFlagSet(showCloudLogs, exitOnRunning *bool) *pflag.FlagSet {
```

Comment:
It seems like we don't need pointers for these booleans, don't we? :thinking:

Reply:
same as the other comment, we need pointers here, but the arguments are removed completely in one of the follow-up PRs https://github.com/grafana/k6/pull/2418 because of some further refactoring